api/types/src/convert.rs

```diff
@@ -79,6 +80,24 @@ impl<'a, R: MoveResolverExt + ?Sized> MoveConverter<'a, R> {
         self.inner.view_resource(typ, bytes)?.try_into()
     }
 
+    pub fn try_into_resources_from_resource_group(
+        &self,
+        bytes: &[u8],
+    ) -> Result<Vec<MoveResource>> {
+        // Resource groups cannot be nested.
```

Comment:
Resource groups cannot be nested because they can only live as a top level resource

Reply:
Removed the comment based on what Daniel/David said.